test/server/kill-test.js

```diff
@@ -0,0 +1,136 @@
+const __importDefault = this && this.__importDefault || function (mod) {
+    return mod && mod.__esModule ? mod : { 'default': mod };
+};
+const sinon     = require('sinon');
+const assert    = require('assert');
+const osFamily = __importDefault(require('os-family'));
```

Comment:
`__importDefault` is an auto-generated construct produced by compiler. Just use `require`.